src/persistence/historykeeper.cpp

```diff
@@ -524,7 +524,7 @@ void HistoryKeeper::removeAvatar(const QString& ownerId)
 bool HistoryKeeper::hasAvatar(const QString& ownerId)
 {
     QSqlQuery sqlAnswer = db->exec(QString("SELECT avatar FROM aliases WHERE user_id= '%1'").arg(ownerId.left(64)));
-    if (sqlAnswer.first() && sqlAnswer.value(0).toByteArray()!=NULL)
+    if (sqlAnswer.first() && (sqlAnswer.value(0).toByteArray().length() > 0) )
```

Comment:
Should simply work with `if (!sqlAnswer.isNull(0))…`. Why always so complicated? :)